.github/workflows/cli.yml

```diff
@@ -13,6 +13,18 @@ jobs:
   build:
     runs-on: ${{ matrix.os }}
 
+    services:
+      postgres:
+        image: postgres:10.8
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: postgres
+        ports:
+          - 5432:5432
```

Comment:
Can you skip the public port and leave it dynamic to avoid port collisions? You do fetch it dynamically below

Reply:
Could you please point me towards how to do that, sounds very reasonable :+1:

Reply:
Not sure about how it works in this particular config format and whatnot, but you should be able to specify just a single number (5432) which means that the other one will be a random free port... If it works like the normal docker.